ci/run-k8s-e2e-tests.sh

```diff
@@ -138,6 +140,10 @@ case $key in
     SYSTEMD_LOGS_IMAGE="$2"
     shift 2
     ;;
+    --conformance-image)
```

Comment:
Why do we need to update the conformance image? it's because of Kind node image change?

Reply:
Sonobuoy will detect Kubernetes version and choose the corresponding image. But if we use "latest", the version detected by Sonobuoy is a dirty version like `v1.30.0-rc-abc`, and the conformance test image does not exist, so I added this argument.